src/pages/en/tutorial/2-pages/5.md

```diff
@@ -45,6 +45,7 @@ There are a few ways to define styles **globally** in Astro, but in this tutoria
     h1 {
       margin: 0;
       padding: 1.25rem 0;
+      color: blue;
```

Comment:
Ah, I removed this intentionally in #1920 but am now seeing the mention of “blue” later in the page:

> Are there any conflicting styles, and if so, which are applied?
>
> Yes, `h1` is defined as blue globally, but purple locally in the `<style>` tag. The purple color is applied.

Let’s remove this fix here and I’ll have a think about how to make sure we still have a some conflicting styles we can ask readers about.

```suggestion
```